tools/flaky-test-reporter/main.go

```diff
@@ -30,14 +30,20 @@ import (
 
 func main() {
 	serviceAccount := flag.String("service-account", os.Getenv("GOOGLE_APPLICATION_CREDENTIALS"), "JSON key file for service account to use")
```

Comment:
update the desc to indicate that this if for GCS auth

Reply:
done